src/index.js

```diff
@@ -0,0 +1 @@
+alert("hello from webpack!");
```

Comment:
Probably don't want to keep this.

Reply:
> I'm assuming this is in progress rather than ready for comments... realised that after I started looking.

yeah, my bad, I should have opened it as a draft PR.